hack/ramen-must-gather

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+function usage {
+	echo "ramen-must-gather [appNamespace [hubClusterName [managedClusterNames...]]]"
+	echo "default appNamespace is current namespace"
+	echo "default hubClusterName is 'hub'"
+	echo "default managedClusterNames are all clusters in your kubeconfig"
+	echo
+	echo "Output: A .tar.gz file containing the gathered cluster data"
+}
+
+# Compute default command line parameters
+# Default namespace is the current namespace
+DefaultNamespace=$(kubectl config view --minify \
+	-ojsonpath='{.contexts[0].context.namespace}')
+# Default clusters are all clusters in the kubeconfig
+DefaultManagedClusters=$(kubectl config view \
+	-ojsonpath='{.contexts[*].context.cluster}')
+
+# Process the command line arguments
+AppNs=${1-"${DefaultNamespace}"}; shift
+HubCluster=${1-"hub"}; shift
+ManagedClusters=( "${*-"${DefaultManagedClusters}"}" )
+
+# CRs to gather in hub
+HubClusterResources=( subscriptions drpc placementrules )
+
+# CRs to gather in managed clusters
+ManagedClusterResources=( subscriptions vrg vr pvc pv pod )
+
+# List of 'namespace;deploymentName;podName' to collect logs
+# in the hub cluster.
+HubClusterLogList=(
+		'ramen-system;ramen-hub-operator;manager'
```

Comment:
ramen-system might be the default namespace, but how about others? IOW, this alone will not work if Ramen is deployed on a different namespace?